### PR TITLE
Remove pdf preview todo

### PR DESCRIPTION
**Pull request by ryankortbeek**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -34,15 +34,15 @@
...
+    t.bigint 'blob_id', null: false
     t.string 'variation_digest', null: false
     t.index %w[blob_id variation_digest], name: 'index_active_storage_variant_records_uniqueness', unique: true
   end
 
   create_table 'diffs', force: :cascade do |t|
     t.integer 'layer_id', null: false
     t.integer 'seq'
-    t.json 'data', default: '{}', null: false
+    t.json 'data', default: '""{}""', null: false
```
Comment (nayancprakash) [inline db/schema.rb]: dont think we wanna keep these changes
```suggestion
    t.json 'data', default: '{}', null: false
```

**Review by nayancprakash — CHANGES_REQUESTED** [review]
One small requested change, otherwise LGTM

File: db/schema.rb (hunk 1 above)
Comment (ryankortbeek) [reply db/schema.rb]: Oh ya good catch

**Review by nayancprakash — APPROVED** [review]
👑

**Pull request merged**